docs/tools/libzbc.md

```diff
@@ -145,18 +146,19 @@ development and tests.
 
 </center>
 
-All functions will behave in the same manner regardless of the type of disk
-being used. The only exception to this principle is the `zbc_errno()` function
-inability to report detailed error information when the zoned block device
-driver is used. The reason for this is that the kernel I/O stack does not have
-the ability to propagate up to the application the detailed information provided
-in command sense data with failed commands.
+All functions behave in the same manner regardless of the type of disk that is
+used. The only exception to this is the `zbc_errno()` function's inability to
+report detailed error information when the zoned block device driver is used.
+The reason for this is that the kernel I/O stack does not have the ability to
+propagate up to the application the detailed information that is provided in
+command sense data [TODO: Damien, what is "command sense data"?] with failed
```

Comment:
SCSI and ATA devices report errors details using "sense data". This is a well-known term for SCSI/ATA devices. So maybe change this to "...that is provided in SCSI/ATA sense data for failed commands."

Reply:
<montgomery burns>Excellent.</montgomery burns>